### PR TITLE
Compare mr results to base commit

### PR DESCRIPTION
**Pull request by tolry**

### CODE REVIEW

File: src/AppBundle/Command/EventMergeRequestCommand.php [hunk 1]
```diff
@@ -16,7 +16,10 @@ protected function configure()
         $this
             ->setName('simpspector:event:merge-request')
             ->addOption('url', null, InputOption::VALUE_OPTIONAL, 'repository url', null)
-            ->addOption('commit', null, InputOption::VALUE_OPTIONAL, 'commit-hash of last commit', '');
+            ->addOption('commit', null, InputOption::VALUE_OPTIONAL, 'commit-hash of last commit', '')
+            ->addOption('from-branch', null, InputOption::VALUE_OPTIONAL, '', '')
+            ->addOption('to-branch', null, InputOption::VALUE_OPTIONAL, '', '')
```
Comment (larsborn) [inline src/AppBundle/Command/EventMergeRequestCommand.php]: may add $description for this two switches?

File: src/AppBundle/Command/JobRunCommand.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $commitHandler->handle($commit);
         }
 
-        if (!$input->getOption('no-garbage-collector')) {
+        if (!!$input->getOption('no-garbage-collector')) {
```
Comment (larsborn) [inline src/AppBundle/Command/JobRunCommand.php]: sure?

File: src/AppBundle/Controller/CommitController.php [hunk 3]
```diff
@@ -75,8 +75,18 @@ public function logAction(Commit $commit)
      *
      * @return array
      */
-    public function diffAction(Commit $from, Commit $to)
+    public function diffAction($from, $to)
```
Comment (larsborn) [inline src/AppBundle/Controller/CommitController.php]: can you also please update the docstring?

File: src/AppBundle/Worker/EventFactory.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+use DavidBadura\GitWebhooks\Struct\Repository;
+use DavidBadura\GitWebhooks\Struct\User;
+use GitElephant\Repository as GitRepository;
+
+class EventFactory
+{
+
+    public function __construct()
+    {
+    }
```
Comment (larsborn) [inline src/AppBundle/Worker/EventFactory.php]: remove maybe?

File: src/AppBundle/Controller/CommitController.php (hunk 3 above)
Comment (tolry) [inline src/AppBundle/Controller/CommitController.php]: done

File: src/AppBundle/Worker/WorkspaceManager.php [hunk 5]
```diff
@@ -44,7 +45,7 @@ public function create(Commit $commit, AbstractLogger $logger = null)
     {
         $logger = $logger ?: new NullLogger();
 
-        $this->cleanUp($commit);
+        //$this->cleanUp($commit);
```
Comment (larsborn) [inline src/AppBundle/Worker/WorkspaceManager.php]: really?

**Comment by DavidBadura** [comment]
:+1:

**Comment by dragonito** [comment]
+1

**Pull request merged**